doc/generating_sphinx_docs.md

```diff
@@ -3,12 +3,14 @@
 ## Set up environment ##
 
 1. Inside a Python virtual environment, run "pip install sphinx==1.6.7"
-2. If Az isn't set up in this virtual environment, run "python scripts\dev_setup.py" from azure-cli
+2. If Az isn't set up in this virtual environment:
+ * pip install azdev
+ * azdev setup -c
```

Comment:
`azdev setup -c` has prerequisites. The more general form would be `azdev setup -c <CLI_REPO_PATH>`

Reply:
Should I update this change to that?

Reply:
I would recommend so yes since using the `-c` form without anything is a special case with prerequisites for it to work.